incident.py

```diff
@@ -6,7 +6,7 @@
 @roles('class-cache')
 def fail_to_mirror():
     """Fails the site to the mirror"""
-    puppet.disable()
+    puppet.disable("Fabric fail_to_mirror task invoked")
```

Comment:
The username gets added automatically: https://github.com/alphagov/fabric-scripts/blob/5496e31efe60539da16e68b489a63351c3ee9127/puppet.py#L17-L20